conduct.md

```diff
@@ -225,19 +225,19 @@ team leaders, technical leads, or the CEO directly.
     those of the team as a whole.
     * Disregard on a conflict that has not yet been discussed or resolved
     to prevent even further conflicts to arise.
-  * Team Leads shall NOT:
+  * Team leads shall NOT:
     * Disregard any conflict that has been brought to their attention.
     * Use their position/ influence to make any type of decision that is not
     beneficial to the team as a whole.
 
 ## Infractions
 
-An ***infraction*** shall hereby be defined as a penalty given as a result of a
+An infraction shall hereby be defined as a penalty given as a result of a
```

Comment:
I think the emphasis here is useful, however it is not necessary as you have noted.

Reply:
Would you prefer it be added back in?

Reply:
It did not match the style of the rest of the document.

Reply:
It is up to you, I think not matching the style is fine justification for removing it.

Reply:
YAY! :sparkles:

Reply:
@Michionlion Do I have your approval with this change? If yes I will be able to merge. If not I would be happy to make any changes to satisfy your demands.